conans/client/conf/__init__.py

```diff
@@ -83,6 +83,7 @@
 default_profile = %s
 compression_level = 9                 # environment CONAN_COMPRESSION_LEVEL
 sysrequires_sudo = True               # environment CONAN_SYSREQUIRES_SUDO
+sysrequires_mode = enabled            # environment CONAN_SYSREQUIRES_MODE (allowed modes enabled/verify/disabled)
```

Comment:
To make sure everything works as expected and backward compat, leave ``sysrequires_mode`` commented out. The default mode, being it not defined should still be good

Reply:
@memsharded Do you mean comment out and launch tests and then put it back again?

Reply:
Nop, just leave it commented out. That will be the default conan.conf in users installation, so we are sure the test suite passes with that default.

BTW, this variable usage is not tested in this PR, probably it should.

Reply:
Ok, commenting it.

About testing not sure if we are talking about the same kind of test but there is one added in this PR:

https://github.com/conan-io/conan/pull/2493/files#diff-aa9f4200e761c7fa62364de5bff87da5R474

checking ``enabled`` for this new env variable ``CONAN_SYSREQUIRES_MODE`` (default mode/behaviour)

Reply:
Not really. Unless I am missing something, no test is actually setting some value for it in *conan.conf* and proving that such value is used. Not saying it is a critical test or that this shouldn't be approved without it, but such behavior is not covered in current tests.